### DISCUSSION TITLE
24 ottobre 2022 - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#24] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/964372c3-bf43-4e22-bfe8-d650e1a13054)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/97389762/a7ccacf3-679d-450e-bc7e-0c9a8dd44431)
Mi sto ancora esercitando quindi potrebbero esserci errori.

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/bcfe0291-1aa3-42cb-bb14-8cc3d7cee7e7)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/33154b59-2348-4c9d-ad30-f7b18df4418b)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/5bb639ed-a373-42be-91f7-e3af921db2e4)

**Reply by salicius** [reply]
Nell'esercizio 2c, ci sono alcune cose che non comprendo:

---
1- Hai scritto (proprio sotto la scritta step2) dopo il calcolo delle coperture di A e quella di B che non esiste D, mentre dopo quella di H che non esiste E, cosa c'entrano D ed E in questo caso ?


---
<img width="796" alt="Screenshot 2025-06-02 at 11 31 50 AM" src="https://github.com/user-attachments/assets/c45d8c58-16b8-492d-b1df-63d871a80e92" />

2- Perché hai scritto queste prima di passare a calcolare F2?

---
3- Hai scritto che ABH-->E si può rimpiazzare con BH-->E, ma siamo sicuri? 
Non c'e E nella chiusura di BH quando testo ABH-->E, vedere:
![450131282-1a2064b2-169d-426c-b699-63d6842957b1](https://github.com/user-attachments/assets/7315f83a-8973-48c7-a672-d1897eb79759)
Dimmi se è corretto

Stessa cosa per la chiusura BH quando si testa BH-->A che a me viene BHDC mentre a te BHEDC
